hydromt/models/model.py

```diff
@@ -1647,131 +1377,12 @@ def read_nc(
-            return self.staticmaps.raster.dims
-
-    @property
-    def coords(self) -> Dict:
-        """Returns the coordinates of model staticmaps.
-
-        ..NOTE: will be deprecated in future versions.
-        """
-        if len(self.staticmaps) > 0:
-            return self.staticmaps.raster.coords
-
-    @property
-    def res(self) -> Tuple:
-        """Returns the resolution of the model staticmaps.
-
-        ..NOTE: will be deprecated in future versions.
-        """
-        if len(self.staticmaps) > 0:
-            return self.staticmaps.raster.res
-
-    @property
-    def transform(self):
-        """Returns the geospatial transform of the model staticmaps.
-
-        ..NOTE: will be deprecated in future versions.
-        """
-        if len(self.staticmaps) > 0:
-            return self.staticmaps.raster.transform
-
-    @property
-    def width(self):
-        """Returns the width of the model staticmaps.
-
-        ..NOTE: will be deprecated in future versions.
-        """
-        if len(self.staticmaps) > 0:
-            return self.staticmaps.raster.width
-
-    @property
-    def height(self):
-        """Returns the height of the model staticmaps.
-
-        ..NOTE: will be deprecated in future versions.
-        """
-        if len(self.staticmaps) > 0:
-            return self.staticmaps.raster.height
-
-    @property
-    def shape(self) -> Tuple:
-        """Returns the shape of the model staticmaps.
-
-        ..NOTE: will be deprecated in future versions.
-        """
-        if len(self.staticmaps) > 0:
-            return self.staticmaps.raster.shape
-
-    @property
-    def bounds(self) -> Tuple:
-        """Returns the bounding box of the model region.
-
-        ..NOTE: will be deprecated in future versions.
-        """
-        if len(self.staticmaps) > 0:
-            return self.staticmaps.raster.bounds
-        else:
-            return self.region.bounds
+        return self.region.crs
 
     # test methods
-    def test_model_api(self):
-        """Test compliance with HydroMT Model API."""
-        warnings.warn(
-            '"test_model_api" is now part of the internal API, use "_test_model_api"'
-            " instead.",
-            DeprecationWarning,
-            stacklevel=2,
-        )
-        return self._test_model_api()
-
-    def _test_model_api(self) -> List:
-        """Test compliance with HydroMT Model API.
-
-        Returns
-        -------
-        non_compliant: list
-            List of model components that are non-compliant with the model API
-            structure.
-        """
-        non_compliant = []
-        for component, dtype in self.api.items():
-            obj = getattr(self, component, None)
-            try:
-                assert obj is not None, component
-                _assert_isinstance(obj, dtype, component)
-            except AssertionError as err:
-                non_compliant.append(str(err))
-
-        return non_compliant
```

Comment:
Did we replace this test somewhere else? I think a lot of plugins use this

Reply:
As far as I can tell this was used to make sure that components had the correct types but this is now checked by the component architecture so it should not be necessary anymore.

Reply:
Ah yes good point!